README.md

```diff
@@ -1,2 +1,46 @@
-# hello-microverse
-first linter usage
+
+
+# Project hello-microverse
+
+> Description the project.
+Added index.html  h1 title 
+style.css style background and style to the title
+test.md for the comment
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+This project adds a style to h1 html elemet.
+
+## Authors
+
+👤 Pamphile
+
+- GitHub: [@pamphilemkp](https://github.com/pamphilemkp)
+- Twitter: [@PamphileMusonda](https://twitter.com/PamphileMusonda)
+- LinkedIn: [Panphile Musonda](https://linkedin.com/in/PanphileMusonda)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] As the previous reviewer suggested. Please kindly create a `MIT.md`  in your root directory so that this MIT link goes to that file.
- copy and paste the following code into your MIT.md and adapt it to your application:
```
## Copyright 2021, [YOUR NAME]

###### Please delete this line and the next one
###### APP TYPE can be a webpage/website, a web app, a software and so on

Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].

THE [APP TYPE] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].
```